site/themes/tailwind-audio.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
I would be nice if these were stored next to the themes, yeah? Not a high priority.

Reply:
yea no, this is intentional. 

`site/themes` hold the site content.
`[root]/themes` are the player.style theme packages (NPM).